src/rapidhtml/app.py

```diff
@@ -20,17 +47,26 @@ class RapidHTML(Starlette):
         - HTMX
     """
 
-    def __init__(self, *args, html_head: typing.Iterable = None, **kwargs) -> None:
+    def __init__(
+        self, *args, html_head: typing.Iterable = None, reload: bool = False, **kwargs
+    ) -> None:
         super().__init__(*args, **kwargs)
-        if not html_head:
-            html_head = ()
+
+        self.reload = reload
         self.html_head = (Script(src="https://unpkg.com/htmx.org@2.0.1"),) + tuple(
-            html_head
+            html_head or ()
         )
-        self.router = RapidHTMLRouter(html_head=self.html_head)
+
+        if reload:
+            self.html_head += (Script(JS_RELOAD_SCRIPT),)
+            self.router = RapidHTMLRouter(html_head=self.html_head)
+            self.router.add_websocket_route("/live-reload", _ReloadSocket)
+        else:
+            self.router = RapidHTMLRouter(html_head=self.html_head)
 
     def serve(self, *args, **kwargs):
```

Comment:
It's possible for the user to pass `reload=` here as well. We should pop it out and raise a Warning.